src/shogun/regression/svr/LibLinearRegression.cpp

```diff
@@ -43,11 +45,18 @@ void CLibLinearRegression::init_defaults()
 
 void CLibLinearRegression::register_parameters()
 {
-	SG_ADD(&m_C, "m_C", "regularization constant",ParameterProperties::HYPER);
-	SG_ADD(&m_epsilon, "m_epsilon", "tolerance epsilon");
-	SG_ADD(&m_epsilon, "m_tube_epsilon", "svr tube epsilon",ParameterProperties::HYPER);
-	SG_ADD(&m_max_iter, "m_max_iter", "max number of iterations");
-	SG_ADD(&m_use_bias, "m_use_bias", "indicates whether bias should be used");
+	SG_ADD(&m_C, "C", "regularization constant", ParameterProperties::HYPER);
+	SG_ADD(
+		&m_epsilon, "m_tube_epsilon", "svr tube epsilon",
+		ParameterProperties::HYPER);
+	SG_ADD(&m_max_iter, "max_iter", "max number of iterations");
+	SG_ADD(&m_use_bias, "use_bias", "indicates whether bias should be used");
+	SG_ADD(
+		(machine_int_t*)&m_liblinear_regression_type,
+		"liblinear_regression_type", "Type of LibLinear regression.");
+	SG_ADD_OPTION("liblinear_regression_type", L2R_L2LOSS_SVR);
+    SG_ADD_OPTION("liblinear_regression_type", L2R_L1LOSS_SVR_DUAL);
```

Comment:
whitespace going wrong here

Reply:
sorry about this, but what about actually merging all those lines into one with a multi-argument macro?
`SG_ADD_OPTION("liblinear_regression_type", L2R_L2LOSS_SVR, L2R_L1LOSS_SVR_DUAL)`

Reply:
I am not sure that would work though. Because you would then have a recursive macro, which then needs to be manually coded, i.e. `SG_ADD_OPTION_1`, `SG_ADD_OPTION_2`, `SG_ADD_OPTION_3` right?